spec/mailers/user_mailer_spec.rb

```diff
@@ -29,7 +29,7 @@
     end
 
     it 'assigns confirmation_url for email body' do
-      @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
+      @mail.body.encoded.should match("https://localhost:3000/g/#{@group.key}")
```

Comment:
assuming you mean I should change `config/environments/test.rb` so that it has the same line as production : 

```ruby
 protocol: ENV['FORCE_SSL'] ? 'https' : 'http'
```

then your proposed method doesn't work. I've tried it and you can change that env, but the app configuration is set at initializations (i think).

I started down a path of overwriting / stubbing the config (was looking at Loomio::Application.config.force_ssl , but that's the wrong thing)

found something. posting it up, but it doesn't feel pretty